options.go

```diff
@@ -192,6 +192,14 @@ func Help(help HelpPrinter) Option {
 	})
 }
 
+// ShortHelp printer to use.
```

Comment:
Can you expand this comment to explain the intent of this and when it would be used?